doc/getting-started.rst

```diff
@@ -754,10 +740,7 @@ details, as well as let you know about features not touched on here.
    rules.
 
 .. [#requirements-one-per-line] Requirements can have whitespace
-   characters as in ``ZEO <=5``, so they're separated by newlines.
-
-.. [#gcc] Currently, this example requires the ability to build
-   Python extensions and requires access to development tools.
+   characters as in ``bobo <3``, so they're separated by newlines.
```

Comment:
Not consciously :)  If I'd noticed it, I would have fixed the usage above too. :)